src/main/resources/assets/waila/lang/de_DE.lang

```diff
@@ -69,16 +77,14 @@ option.vanilla.leverstate=Hebelstatus
 option.vanilla.repeater=Repeater-Verzögerung
 option.vanilla.comparator=Komparatormodus
 option.vanilla.redstone=Redstone-Stärke
-option.vanilla.alternatecropitem=Wechselnde Ernte
-option.vanilla.jukebox=Musikbox-Platte
+option.vanilla.jukebox=Plattenspieler-Schallplatte
 option.vanilla.silverfish=Silberfisch verdecken
-option.vanilla.furnacedisplay=Zeige Ofen Fortschritt
+option.vanilla.furnacedisplay=Zeige Fortschritt des Ofens
 option.general.showents=Zeige Wesen
-option.general.showhp=Zeige Gesundheit von Lebewesen
+option.general.showhp=Zeige Gesundheit des Lebewesens
```

Comment:
Was the former translations not better? It's less specific.

Reply:
Yes, in that case I think...